test/models/notification_test.rb

```diff
@@ -78,12 +78,15 @@ class NotificationTest < ActiveSupport::TestCase
       notification.subscriber_ids
   end
 
-  test 'Admin notifications should subscribe to all admin users' do
+  test 'Admin notifications should subscribe to all admin users except hidden' do
     notification = FactoryGirl.build(:notification,
                                      :audience => Notification::AUDIENCE_ADMIN)
-    assert User.only_admin.count > 0
-    assert_equal User.only_admin.reorder('').pluck(:id).sort,
-      notification.subscriber_ids.sort
+    FactoryGirl.create(:user, :admin)
+
+    subscriber_ids = notification.subscriber_ids
+    assert_includes subscriber_ids, admin.id
+    assert_includes subscriber_ids, User.anonymous_admin.id
```

Comment:
I might be reading this wrong, but it sounds like the notification does notify the anonymous_admin(hidden user)? shouldnt it not include it?